azure-pipelines.yml

```diff
@@ -193,12 +199,10 @@ stages:
               python.version: '3.8'
               pandas.version: 'latest'
               GE_pytest_pip_opts: '--requirement requirements-dev.txt --constraint constraints-dev.txt'
-# NOTE - 20210621 - JPC - as of this time, snowflake connector does not install on python 3.9; we are not
```

Comment:
This was fixed recently with https://github.com/great-expectations/great_expectations/pull/3119